core/controllers/base.py

```diff
@@ -440,7 +443,7 @@ def handle_exception(self, exception, unused_debug_mode):
         """Overwrites the default exception handler.
 
         Args:
-            exception: The exception that was thrown.
+            exception: exception. The exception that was thrown.
```

Comment:
I think the class name (and therefore the type) is Exception

Reply:
Done.